sample-apps/deepedit/main.py

```diff
@@ -37,37 +39,39 @@
 
 class MyApp(MONAILabelApp):
     def __init__(self, app_dir, studies, conf):
-        self.network = DynUNetV1(
-            spatial_dims=3,
-            in_channels=3,
-            out_channels=1,
-            kernel_size=[
+        network_params = {
+            "spatial_dims": 3,
+            "in_channels": 3,
+            "out_channels": 1,
+            "kernel_size": [
                 [3, 3, 3],
                 [3, 3, 3],
                 [3, 3, 3],
                 [3, 3, 3],
                 [3, 3, 3],
                 [3, 3, 3],
             ],
-            strides=[
+            "strides": [
                 [1, 1, 1],
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 1],
             ],
-            upsample_kernel_size=[
+            "upsample_kernel_size": [
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 2],
                 [2, 2, 1],
             ],
-            norm_name="instance",
-            deep_supervision=False,
-            res_block=True,
-        )
+            "norm_name": "instance",
+            "deep_supervision": False,
+            "res_block": True,
+        }
+        self.network = DynUNetV1(**network_params)
```

Comment:
Is it really necessary to have tow network definitions here? Is it possible to use the same neural net but in `eval` mode in  other places:
e.g you could use
```
methods["EPISTEMIC"] = EpistemicScoring(
                model=[self.pretrained_model, self.final_model],
                network=self.network_with_dropout,
                transforms=self._infers["deepedit_seg"].pre_transforms(),
                num_samples=self.epistemic_samples,
            )
```
in scoring, but
```
def init_infers(self) -> Dict[str, InferTask]:
        return {
            "deepedit": Deepgrow(
                [self.pretrained_model, self.final_model],
                self.network,
                self.network.eval(),
                spatial_size=self.planner.spatial_size,
                target_spacing=self.planner.target_spacing,
            ),
            "Histogram+GraphCut": HistogramBasedGraphCut(),
        }
```